src/iterators/inverted_index_iterator.c

```diff
@@ -10,55 +10,25 @@
 #include "inverted_index_iterator.h"
 #include "redis_index.h"
 
-// pointer to the current block while reading the index
```

Comment:
You can try running the `./tests/cpptests/test_cpp_iterator_index.cpp` to check if/how this affects performance.

Reply:
I get a permission denied with that command. But was able to run this `./bin/linux-x64-release/search-community/micro-benchmarks/benchmark_index_iterator`.

The results are about the same as master. Some are faster and others slower. And these change between different runs.